text/0632-deprecate-string-based-actions.md

```diff
@@ -0,0 +1,237 @@
+- Start Date: 2020-05-23
+- Relevant Team(s): Ember.js
+- RFC PR: https://github.com/emberjs/rfcs/pull/632
+- Tracking: (leave this empty)
+
+# Deprecate String Based Actions
+
+## Motivation
+
+In Ember 1.13 we introduced [closure actions](https://github.com/emberjs/rfcs/blob/00ac2685c86f27d41547012903f485a4ef338d27/active/0000-improved-actions.md), which have been recommended over string based actions. This officially deprecates string based actions in favor of closure actions,
```

Comment:
IMO this is not a motivation for deprecating string based actions, rather it is a motivation for adopting closure actions in new code. I agree with those points, but deprecating functionality like this that doesn't hurt anyone is not "stability without stagnation", it is just creating churn for people with old apps.

My recent experience coming back to an Ember 3.x app (started in 1.12) after a few years and trying to get to 4.x is that this framework has *incredible* amounts of churn because of deprecation like this. It just creates work for anyone with an older app without any stated benefit. I have spent weeks now pointlessly refactoring from one pattern to another simply because it was deprecated, while looking at RFCs there is no actual reason given for the deprecation (outside of, "we have invented another way to do it").

What I would really love to see in this motivation section is some reason why this feature is specifically harmful to keep around. For example, maybe it is complicating the code in a way that makes adding other features harder. That would at least be a compelling motivation to remove it.

PS. I hope I don't come across as unnecessarily harsh, I really do not mean it that way. I hope you understand where I am coming from.